### PR TITLE
Clarify plugin unit testing instructions between Travis and locally

### PR DESCRIPTION
**Pull request by jameselks**
I'm new to unit testing and if I'd known I didn't have to set up a server to generate the test files I wouldn't have bothered. So hopefully this clarifies that there's not much to it if you want to run unit tests on Travis. Let me know if I've missed anything.

### CODE REVIEW

File: plugin-unit-tests.md [hunk 1]
```diff
@@ -1,29 +1,40 @@
...
 
-This guide will demonstrate how to:
+## Running tests on Travis CI
 
-* Set up unit tests for an existing plugin, using WP-CLI
-* Run the tests locally
+If you host your plugin on Github and enable [Travis CI](http://about.travis-ci.org), the tests will be run automatically after every commit you make to the plugin.
 
-We're going to assume that you already have a plugin called `my-plugin`.
+All you need to do to enable this is copy and then edit the following files from the [WP-CLI sample plugin](https://github.com/wp-cli/sample-plugin):
```
Comment (jameselks) [inline plugin-unit-tests.md]: Yes, although to do this wouldn't you need to setup the entire environment locally?

File: plugin-unit-tests.md [hunk 2]
```diff
@@ -1,29 +1,40 @@
...
+1. [Install PHPUnit](https://github.com/sebastianbergmann/phpunit#installation) (5.x is only supported when running php7, phpunit 4.8 is required when running php5).
 
-2) Generate the plugin test files: 
+1. Generate the plugin test files: 
 
-```bash
-wp scaffold plugin-tests my-plugin
-```
+  ```bash
+  wp scaffold plugin-tests my-plugin```
```
Comment (jameselks) [inline plugin-unit-tests.md]: Ah, sorry.

File: plugin-unit-tests.md [hunk 3]
```diff
@@ -1,29 +1,40 @@
...
 
-This command will generate all the files needed for running tests, including a `.travis.yml` file. If you host your plugin on Github and enable [Travis CI](http://about.travis-ci.org), the tests will be run automatically after every commit you make to the plugin.
+This command will generate all the files needed for running tests, including a `.travis.yml` file. 
 
-3) Initialize the testing environment locally: `cd` into the plugin directory and run the install script. (You'll need to already have `svn` and `wget` installed.)
```
Comment (jameselks) [inline plugin-unit-tests.md]: Well, I couldn't see why you'd actually need svn. I'll add back the wget instructions, it should be higher up.

File: plugin-unit-tests.md [hunk 4]
```diff
@@ -1,29 +1,40 @@
 # Plugin Unit Tests
 
-This guide will demonstrate how to:
```
Comment (jameselks) [inline plugin-unit-tests.md]: Done.

**Comment by jameselks** [comment]
Thanks for the comments @danielbachhuber. Have made some changes, and asked some questions above.

This is my first commit to an open source project, so thanks for being so positive :)

File: plugin-unit-tests.md (hunk 3 above)
Comment (danielbachhuber) [inline plugin-unit-tests.md]: > Well, I couldn't see why you'd actually need `svn`.

It's a part of the installation script, here: https://github.com/wp-cli/scaffold-command/blob/master/templates/install-wp-tests.sh#L82-L83

File: plugin-unit-tests.md (hunk 1 above)
Comment (danielbachhuber) [inline plugin-unit-tests.md]: > although to do this wouldn't you need to setup the entire environment locally?

Ah, yes, I suppose that's an implicit assumption (that you have a functioning local dev environment).

**Comment by danielbachhuber** [comment]
> Have made some changes, and asked some questions above.

Great. I went ahead and made a couple minor edits, which you can see in the commits above ^. Looks good to land now.

> This is my first commit to an open source project, so thanks for being so positive :)

Awesome — welcome to the open source world!

**Pull request merged**